src/main/scala/org/constellation/checkpoint/CheckpointBlockValidator.scala

```diff
@@ -185,14 +185,14 @@ class CheckpointBlockValidator[F[_]: Sync](
 
     def validateBalance(address: String, t: Iterable[Transaction]): F[ValidationResult[List[Transaction]]] =
       lookup(address).map { balance =>
-        val amount = t.map(_.amount).sum
+        val amount = t.map(_.amount).map(BigInt(_)).sum
```

Comment:
I am wondering if we should always use BigInt (which is no that efficient) instead of using it only if it overflows 🤔 

But maybe it is microoptimisation which we do not need